signalfx-java/src/main/java/com/signalfx/connection/RetryHandler.java

```diff
@@ -15,8 +15,8 @@
  */
 class RetryHandler extends DefaultHttpRequestRetryHandler {
 
-  public RetryHandler() {
-    super(3, true, Arrays.asList(
+  public RetryHandler(final int maxRetries) {
+    super(maxRetries, true, Arrays.asList(
```

Comment:
Can we include a constructor overload that keeps the existing behavior (3 retries)?